packages/contracts-bedrock/contracts/test/FaultDisputeGame.t.sol

```diff
@@ -279,10 +291,20 @@ contract FaultDisputeGame_Test is DisputeGameFactory_Init {
     }
 
     /**
-     * @dev Static unit test asserting that resolve reverts when the game is not in progress.
+     * @dev Static unit test asserting that resolve reverts when the game state is
+     *      not in progress.
```

Comment:
Style guide is to not use the `@dev` tag (see: https://github.com/ethereum-optimism/optimism/blob/develop/packages/contracts-bedrock/STYLE_GUIDE.md#style)
```suggestion
     * @notice Static unit test asserting that resolve reverts when the game state is
     *      not in progress.
```

Reply:
This is within tests; I think the `@dev` tag is fine here personally.

Reply:
Ok, I think we should then update the style guide - created a [linear ticket](https://linear.app/optimism/issue/CLI-4144/fixctb-style-guide-dev-targ) to track :)